src/libYARP_OS/include/yarp/os/Random.h

```diff
@@ -8,7 +8,6 @@
 #define YARP_OS_RANDOM_H
 
 #include <yarp/os/api.h>
-
```

Comment:
This change is not needed, since this is the only change to the file, please restore the original state of this file